src/OrchardCore/OrchardCore.Infrastructure/Scripting/CommonGeneratorMethods.cs

```diff
@@ -11,7 +11,15 @@ public class CommonGeneratorMethods : IGlobalMethodProvider
         Name = "base64",
         Method = serviceProvider => (Func<string, string>)(encoded =>
         {
-            return Encoding.UTF8.GetString(Convert.FromBase64String(encoded));
+            var neededLength = GetByteArrayLengthFromBase64(encoded);
+            Span<byte> bytes = new byte[neededLength];
```

Comment:
This would be the same as the code on the left. But the code on the left is one line.